index.html

```diff
@@ -28,26 +28,34 @@
   <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap" rel="stylesheet">
 </head>
 <body>
-  <div class="row head-container column-full-header">
+  <div class="row background-header height-header align-header column-full">
```

Comment:
One element can't be both a row and a column, and this row and column is not within a container. You should make this element the child of an element with the container class, and give it a child with the column-full class.